app/components/confirm-existing-instance-modal.hbs

```diff
@@ -0,0 +1,39 @@
+<AuModal @modalOpen={{true}} @closeModal={{this.close}}>
+  <:title>Dit concept werd al gebruikt</:title>
+  <:body>
+    <div>
+      Er bestaat al (minstens) een instantie op basis van dit concept. Vermijd meerdere versies van éénzelfde product of
+      dienst. Zo
+      voorkom je uiteenlopende of verouderde informatie en extra werklast. Is de bestaande instantie niet meer relevant?
+      Verwijder deze dan.
+    </div>
+    <div class="au-u-margin-top">
+      <span class="au-u-h4 au-u-medium">Bestaande instanties</span>
+      {{#each @data.existingInstances as |instance|}}
+      <p>
+        <AuLink class="au-u-margin-right" @route="public-services.details" @model={{instance.id}} @query={{hash
```

Comment:
The `preview=true` doesn't do anything here I think. AFAIK that's only a thing for concepts.